### PR TITLE
setters for tms/pms urls

### PR DESCRIPTION
**Pull request by iamolegga**
Hi, finally I found time for this 🙂

As we have breaking changes here, after merging this pr, the new version should be published with next major. Btw, I was forced to reduce minimal limit of covered branches, because after merge total amount reduced too, but I just could not figure it out why, because new setters are covered 🤔

### CODE REVIEW

File: test/fixtures/specs/issueAndTms.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { suite, test } from 'mocha-typescript';
-import { issue, testCaseId } from '../../../index';
+import { issue, setPmsUrl, setTmsUrl, testCaseId } from '../../../index';
+
+setPmsUrl('http://pms-url');
```
Comment (sskorol) [inline test/fixtures/specs/issueAndTms.ts:4]: I believe you can move these 2 into `before` hook in a class body. It'll be more natural assuming OO-style.

File: index.d.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import { Severity } from 'allure2-js-commons';
+export declare function setPmsUrl(value: string): void;
```
Comment (sskorol) [inline index.d.ts:2]: let's probably call these 2 `assignXXXUrl`

**Comment by sskorol** [comment]
@iamolegga as you haven't responded to comments for a long time, I decided to include them into my PR and close this one. See latest updates, which include allure-js and testdeck migration stuff.

**Comment by iamolegga** [comment]
Sorry, was a bit busy

**Pull request closed without merging**